testdata/md5sum.cwl

```diff
@@ -11,7 +11,7 @@ outputs:
 
 steps:
   md5sum:
-    run: dockstore-tool-md5sum.cwl
+    run: https://raw.githubusercontent.com/common-workflow-language/workflow-service/master/testdata/dockstore-tool-md5sum.cwl
```

Comment:
This looks fine to me.  Should get some feedback from David though on his thoughts.

Reply:
One of the main feature improvements of switching to multipart upload is the ability to send "workflow_attachments":

https://github.com/ga4gh/workflow-execution-service-schemas/blob/develop/openapi/workflow_execution_service.swagger.yaml#L123

The idea is that if you set the content disposition header filename='dockstore-tool-md5sum.cwl' the WES will put it into a temp dir before trying to run anything (just like it does with the actual descriptor).

This change circumvents the need to implement that feature, which is probably not a great idea. It would be a cool feature to demonstrate but can be addressed in a future PR.